requirements.txt

```diff
@@ -5,7 +5,7 @@ boto3>=1.4.4
 cffi>=1.11.5
 configparser>=3.5.0
 croniter>=0.3.16
-elasticsearch>=7.0.0
+elasticsearch>=7.5.0
 envparse>=0.2.0
```

Comment:
If you can revert the es lib upgrade (see #90 for why this is needed) we can get this merged in. If there is any applicable doc updates that might be helpful as well.

Reply:
Nevermind, I was able to revert it.